docs/relational-databases/performance/tune-performance-with-the-query-store.md

```diff
@@ -68,18 +68,25 @@ ORDER BY total_execution_count DESC;
 The number of queries with the longest average execution time within last hour:
 
 ```sql
-SELECT TOP 10 rs.avg_duration, qt.query_sql_text, q.query_id,
-    qt.query_text_id, p.plan_id, GETUTCDATE() AS CurrentUTCTime,
-    rs.last_execution_time
+SELECT TOP 10
```

Comment:
The query below is what you get from the QDS Top Resource Consuming Queries report in SSMS, for Avg Duration.  

SELECT TOP 10
    p.query_id query_id,
    q.object_id object_id,
    ISNULL(OBJECT_NAME(q.object_id),'''') object_name,
    qt.query_sql_text query_sql_text,
    ROUND(CONVERT(float, SUM(rs.avg_duration*rs.count_executions))/NULLIF(SUM(rs.count_executions), 0)*0.001,2) avg_duration,
    SUM(rs.count_executions) count_executions,
    COUNT(distinct p.plan_id) num_plans
FROM sys.query_store_runtime_stats rs
    JOIN sys.query_store_plan p ON p.plan_id = rs.plan_id
    JOIN sys.query_store_query q ON q.query_id = p.query_id
    JOIN sys.query_store_query_text qt ON q.query_text_id = qt.query_text_id
WHERE rs.last_execution_time > DATEADD(hour, -1, GETUTCDATE())
GROUP BY p.query_id, qt.query_sql_text, q.object_id
HAVING COUNT(distinct p.plan_id) >= 1
ORDER BY avg_duration DESC

Notice the way that the average is calculated, this is what should be used.  Doing (AVG(rs.avg_duration)) is an average of averages, which results in a different (and not accurate) number.  Also note that the query below does the math to convert to seconds.  Default in the Query Store objects is milliseconds.  Remove that math to get values consistent with the other queries on the page...

I would adjust the original query to be:

SELECT TOP 10
   ROUND(CONVERT(float, SUM(rs.avg_duration*rs.count_executions))/NULLIF(SUM(rs.count_executions), 0),2) avg_duration,
   SUM(rs.count_executions) AS total_execution_count,
   qt.query_sql_text,
   q.query_id,
   qt.query_text_id,
   p.plan_id,
   GETUTCDATE() AS CurrentUTCTime,
   MAX(rs.last_execution_time) AS last_execution_time
FROM sys.query_store_query_text AS qt
JOIN sys.query_store_query AS q ON
   qt.query_text_id = q.query_text_id
JOIN sys.query_store_plan AS p ON
   q.query_id = p.query_id
JOIN sys.query_store_runtime_stats AS rs ON
   p.plan_id = rs.plan_id
WHERE rs.last_execution_time > DATEADD(HOUR, -1, GETUTCDATE())
GROUP BY qt.query_sql_text, q.query_id, qt.query_text_id, p.plan_id
ORDER BY AVG(rs.avg_duration) DESC;